lizard_auth_server/forms.py

```diff
@@ -42,6 +46,40 @@ def clean(self):
         return new_data
 
 
+class JWTDecryptForm(forms.Form):
+    key = forms.CharField(max_length=1024)
+    message = forms.CharField(max_length=8192)
+
+    def clean(self):
+        """Verifies the JWT from the site and cleans form data.
+
+        Note: replaces the original form data with JWT payload, which should
+        contain a dictionary with the following keys:
+
+        ['key',
+         'domain',
+         'force_sso_login',
```

Comment:
Klinkt alsof deze nu verplicht is. Dat is voor m'n gevoel niet zo lekker. Voor JWT zou ik dan toch liever een alternatieve URL naast die voor de verplichte inlog hebben. 

Die JWT message zou zo klein mogelijk moeten zijn, lijkt me. Daarom vind ik het wat raar om if/else achtige zaken er bij in te stoppen. Qua ontwerp.